pro/src/utils/trimStringsInObject.ts

```diff
@@ -0,0 +1,19 @@
+import { isPlainObject } from './isPlainObject'
+
+export const trimStringsInObject = (obj: any, visited = new Set()) => {
```

Comment:
Est-ce qu'on ne voudrait pas que des `object` ?
```suggestion
export const trimStringsInObject = (obj: object, visited = new Set()) => {
```

Reply:
Bien vu. C'est un oubli de précision après que j'ai démarré la fonction.
J'ai mis un `Record<string, any>` afin de pouvoir indexer avec `obj[key]` (commit incoming …)